### ISSUE TITLE
Faster conditions UX during Signup dialog

### ISSUE THREAD

**Issue by tassoman** [syuilo/Misskey#11737]
## 💡 Summary

When you register a new account, you're required to agree 3 steps.

## 🥰 Expected Behavior

Only after you agreed the step, the accordion UI should compress itself showing the green :heavy_check_mark: mark

## 🤬 Actual Behavior

Actually when you agree, you must compress the sections yourself and discover the 3 marks :heavy_check_mark: :heavy_check_mark: :heavy_check_mark: 

## 📝 Steps to Reproduce

1. open misskey into a new anonymous navigation
2. click button: signup on this instance
3. read rules, scroll the modal if needed
4. click button: accept
5. see: accordion did not compressed
6. click accordion up arrow :arrow_up_small: and compress the accordion
7. repeat for the other agreements
8. finally see all three steps done with :heavy_check_mark:

### 💻 Frontend
* Model and OS of the device(s): any
* Browser: any
* Server URL: /
* Misskey: 13.14.2

### 🛰 Backend (for server admin)

* Installation Method or Hosting Service: develop branch, docker compose
* Misskey: 13.14.2
* Node: 20.x.x
* PostgreSQL: 15.x.x
* Redis: 7.x.x
* OS and Architecture: docker

**Comment by kakkokari-gtyih** [comment]
I don't think we have to automatically close the accordion UI just because you agree with it. Even if you don't close the accordion, you can still press the 'Next' button if you agree with all the items (Sorry if I've misunderstood your concern).

**Comment by tassoman** [comment]
You already put a green flag on agreed accordion. Why don't compress, also?

If you can automatically compress accordion when is done :heavy_check_mark:  continue button "magically" appears.  
Otherwise, continue button is always outside the screen. I recorded a video.

[Kazam_screencast_00042.webm](https://github.com/misskey-dev/misskey/assets/92727/67b907db-686c-4f6a-b967-9c7879ee2b1a)
